File: php/testcode/fileinclusion_011.php
<?php
require_once __DIR__ . '/shared.php';

// vuln-code-snippet start php_fi_remote_url
function fileinclusion011(BenchmarkRequest $req): BenchmarkResponse {
    $url = $req->param('url');
    include($url); // vuln-code-snippet vuln-line php_fi_remote_url
    return BenchmarkResponse::ok("remote content loaded");
}
// vuln-code-snippet end php_fi_remote_url

File: ruby/testcode/fileinclusion_011.rb
require_relative 'shared'

# vuln-code-snippet start ruby_fi_eval_file
def eval_plugin(req)
  path = req.param('path')
  eval(File.read(path)) # vuln-code-snippet vuln-line ruby_fi_eval_file
  BenchmarkResponse.ok('loaded')
end
# vuln-code-snippet end ruby_fi_eval_file
